src/policy/policy.cpp

```diff
@@ -169,6 +169,15 @@ bool IsStandardTx(const CTransaction& tx, const kernel::MemPoolOptions& opts, st
             }
         }
 
+        if (whichType == TxoutType::WITNESS_V0_SCRIPTHASH && opts.reject_tokens) {
```

Comment:
If I'm reading the linked doc correctly, the "STAMP:" part is in an OP_RETURN, not p2wsh?